vidarr-server/src/main/resources/db/migration/V0012__add_get_ids_for_downstream_workflow_runs_function.sql

```diff
@@ -0,0 +1,13 @@
+-- The input file ids are Vidarr IDs formatted like: vidarr:<server>/file/<hash>
+-- Unnest the array of Vidarr IDs, then split each Vidarr ID on the slash and select the third item (<hash>)
+
+    CREATE OR REPLACE FUNCTION get_ids_for_downstream_workflow_runs(wr_ids BIGINT[]) RETURNS table (wfr_id bigint) AS $$
+      BEGIN
+        RETURN QUERY
+        SELECT DISTINCT id FROM workflow_run
+                WHERE array_length(input_file_ids, 1) > 0 AND
+                  -- workflow uses files produced by target workflow run as its input
+                  (SELECT ARRAY(SELECT DISTINCT SPLIT_PART(UNNEST(input_file_ids), '/', 3)::VARCHAR FROM workflow_run)) <@ (
```

Comment:
Migration workflow runs currently do not use the `vidarr:<server>/file/<hash>` input id format. Either the ids need to be updated to use this format or coalescing on split_part and the unmodified hash id, something like:
```
select coalesce(nullif(vidarr_id,''), nullif(migration_id,'')) input_id, hash_id from (select split_part(unnest(input_file_ids),'/',3) vidarr_id, unnest(input_file_ids) migration_id, hash_id FROM workflow_run where cardinality(input_file_ids) > 0) as a;
```

Reply:
Heather found that this was fixed and that migration in production is producing the expected input file pattern (vidarr:_/file/<input_file_hash_id>).